frontend/test/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file is part of the ALICE Electronic Logbook v2, also known as Jiskefet.
+ * Copyright (C) 2020  Stichting Hogeschool van Amsterdam
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/* eslint-disable */
+
+const assert = require('assert')
+const puppeteer = require('puppeteer')
+const path = require('path');
+const { HttpServer } = require('@aliceo2/web-ui');
+
+const PORT = 3001
+
+describe('Frontend', () => {
+    before(() => {
+      http = new HttpServer({ port: PORT });
```

Comment:
Shouldn't this be the same kind of construction as the backend, rebuilding the server.js doesn't seem right to me.